### ISSUE TITLE
500 Error for Faceted Search

### ISSUE THREAD

**Issue by Mahir-Sparkess** [cedadev/stac-fastapi-elasticsearch#12]
On the POST search endpoint, doing a facet search returns a 500 Error.
## The request body
```JSON
{
 "filter": {
    "institution_id": ["CNRM-CERF"]
  }
}
```
## The response body
Error: the response status is 500
```JSON
{
  "detail": "Failed to evaluate node of type <class 'NoneType'>"
}
```

**Comment by Mahir-Sparkess** [comment]
@rsmith013 I don't have the cog to assign someone to an issue... So here's a tag.

**Comment by Mahir-Sparkess** [comment]
Unsure if this is relevant but in the response body for Queryables, the properties are empty:

## Response Body

```JSON
{
  "$schema": "https://json-schema.org/draft/2019-09/schema",
  "$id": "https://example.org/queryables",
  "type": "object",
  "title": "Queryables for Example STAC API",
  "description": "Queryable names for the example STAC API Item Search filter.",
  "properties": {}
}
```
where if I do a POST search and `filter={}` it returns a 200 response and completes a POST request with what other params I may have.

**Comment by rsmith013** [comment]
Error due to badly formatted filter request but server should return 400 not 500. 
Fixed in 823eb2e
